video/out/gpu_next/context.c

```diff
@@ -187,6 +187,11 @@ bool gpu_ctx_resize(struct gpu_ctx *ctx, int w, int h)
         // vulkan RA handles this by itself
         return true;
 #endif
+#if HAVE_GL
+    if (ra_is_gl(ctx->ra_ctx->ra) && (w == 0 || h == 0))
```

Comment:
why allow it for d3d11? I assume libplacebo prints this error message independent of swapchain backend/type.

Reply:
It's allowed for vulkan. I have no idea about d3d11 but the error returned makes it sound like it's opengl-specific.

Reply:
Right, appears to be allowed for d3d11 too.
But there's no point in passing through zero values to `pl_swapchain_resize` in any case, libplacebo will query the size but it's not like mpv uses the returned values.